test/functional/fixtures/runner/upload/upload-test.js

```diff
@@ -0,0 +1,20 @@
+var expect = require('chai').expect;
+
+
+describe('Upload', function () {
+    it('Should fail when uploading a non-existent file', function () {
+        return runTests('upload.test.js', 'Upload a non-existent file - should fail', { shouldFail: true })
+            .catch(function (err) {
+                expect(err).contains('fake.jpg');
+            });
+    });
+
+    it('Should fail when uploading multiple files including non-existent files', function () {
+        return runTests('upload.test.js', 'Upload multiple files - should fail', { shouldFail: true })
+            .catch(function (err) {
+                expect(err).contains('fake1.jpg');

```

Comment:
BTW, why we use images. Can we use just plain text, so we will not add big unnecessary data to the repo?